File: Semester 2/Research Project/2020_AON01/Stage_API/Stage_API.Domain/Classes/Review.cs
using Stage_API.Domain.enums;
using System;
using System.Runtime.Serialization;

namespace Stage_API.Domain.Classes
{
    public class Review
    {
        public int Id { get; set; }
        public DateTime Date { get; set; }
        public string Text { get; set; }
        public BeoordelingStatus Status { get; set; }
        public BeoordelingStatus StatusVoorstel { get; set; }
        public Reviewer Reviewer { get; set; }
        [IgnoreDataMember]
        public int ReviewerId { get; set; }
        public Stagevoorstel Stagevoorstel { get; set; }
        [IgnoreDataMember]
        public int StagevoorstelId { get; set; }
    }
}

File: Semester 2/Research Project/2020_AON01/Stage-Frontend/src/app/classes/review.ts
import { Stagevoorstel } from './stagevoorstel';
import { BeoordelingStatus } from './enums/beoordeling-status.enum';
import { Reviewer } from './reviewer';

export class Review {
  id: number;
  date: string; // DateTime;
  text: string;
  status: BeoordelingStatus;
  voorstelStatus: BeoordelingStatus;
  reviewer: Reviewer;
  stagevoorstel: Stagevoorstel;
}
